### PR TITLE
Fix buggy header navigation bar

### LINKED ISSUE #1055: Fix buggy header navigation bar when resizing from desktop view to mobile view
### Overview
The header navigation bar is buggy when resizing from desktop view to mobile view. 

The issues follow:
- [x] The header navigation bar drops down without clicking the navigation icon.  (when resizing the page from desktop view to mobile view where the viewport width is < 768 px)
- [x] The header navigation bar disappear from page (when the navigation bar is closed by clicking the X icon, and the viewport resize to >= 768px)

#### the header navigation bar drops down with navigation icon when viewport's width is less than 768px.
Screen Shot 2021-02-14 at 9 23 32 PM

#### the header navigation bar remains open after one click on top right navigation icon, now close icon substitutes navigation icon.
Screen Shot 2021-02-14 at 9 23 54 PM

#### after one click on top right X icon
Screen Shot 2021-02-14 at 9 24 11 PM

#### header navigation disappears when resize back to >=768 px
Screen Shot 2021-02-14 at 9 24 29 PM

### Resources
https://www.hackforla.org/about-us

### PR DESCRIPTION
**Pull request by akibrhast**
Fixes #1055 

- Fixes the buggy navigation bar, where it would default to open state in mobile view to now defaulting to being on closed state by default.
- The header navigation bar disappear from page (when the navigation bar is closed by clicking the X icon, and the viewport resize to >= 768px)

### CODE REVIEW

**Comment by qiqicodes** [comment]
#### A1. The navigation bar is still buggy, where it would default to open state in mobile view
https://user-images.githubusercontent.com/68657634/109851802-f19b4700-7c08-11eb-92b9-ba6493511d2e.mov

#### B1. After closing the header navigation, the hamburger navigation icon remains until it is 768px for width. 
#### B2. The header navigation bar disappears from 768 - 975px.
https://user-images.githubusercontent.com/68657634/109851666-cadd1080-7c08-11eb-8356-133d32343bf6.mov

**Comment by akibrhast** [comment]
@qiqicodes It seems to be working fine on my side. Were those videos run before or after cache clearing?
I have tested it out both on chrome and edge.

**Comment by akibrhast** [comment]
@qiqicodes After our zoom screen sharing just now. I have been unable to replicate the issue on my end. I have tried replicating the issue in both chrome and edge whilst setting the height to 962px as your video has it. However I am still unable to replicate what your video shows. I have confirmed that @qiqicodes did clear her cache. @qiqicodes Could you please provide the chrome version that your test in the video ran on?

@drubgrubby @jbubar - Would it be possible for you folks to pull this and try it out on your end. If you are able to replicate the problems as shown in the video. Could you please give details on how you managed to replicate it .. Since I am not able to.


1157_hack4la

**Comment by qiqicodes** [comment]
@akibrhast I'm running the latest Chrome version. I have tried rendering the page again with docker. The navigation bar still drops down.
Screen Shot 2021-03-03 at 11 18 39 AM

**Comment by kevinreber** [comment]
@qiqicodes I also noticed this issue on my machine, but it rarely happens on my end. Tried recreating the issue it on Safari with @akibrhast and didn't have any problems.

I'm working on macOS Catalina

Mac OS

Chrome

File: _includes/hamburger-nav.html [hunk 1]
```diff
@@ -1,91 +1,55 @@
 <div id="swapButton">
   <ul class="inline-list">
     <li>
-      <button onclick="burgerSwap()" id="burgerImage">{% include svg/icon-hamburger-nav.svg %}</button>
-      <button onclick="burgerSwap()" id="burgerImageX">{% include svg/icon-hamburger-nav-x.svg %}</button> 
+      <button id="burgerImage" >{% include svg/icon-hamburger-nav.svg %}</button>
```
Comment (kevinreber) [inline _includes/hamburger-nav.html:4]: Why did you remove the second onclick?

**Review by kevinreber — APPROVED** [review]
Looks good! Works on my machine

File: _includes/hamburger-nav.html (hunk 1 above)
Comment (akibrhast) [reply _includes/hamburger-nav.html:4]: So the reason I removed the second onclick, was because having both those elements on top of each other was causing an issue in the drop down where the first click would not neccearily close/open the drop down. It will only do so after the second click. I believe this was because of the existence of the two onlcick functions on top of each other.

**Review by jbubar — APPROVED** [review]
@akibrhast this is great!
It fixed a few bugs with the nav!
and the code is simpler too.
